internal/kuberesource/resourcegen/main.go

```diff
@@ -42,18 +42,30 @@ func main() {
 		var err error
 		switch set {
 		case "coordinator":
-			subResources = kuberesource.CoordinatorBundle()
+			subResources, err = kuberesource.CoordinatorBundle(platform)
+			if err != nil {
+				log.Fatalf("Error generating coordinator: %v", err)
+			}
```

Comment:
There's an error handler outside the switch statement, would be enough to rely on that.